app/components/viral/pill_component.erb

```diff
@@ -0,0 +1,3 @@
+<%= render(Viral::BaseComponent.new(tag: 'span', **system_arguments)) do %>
```

Comment:
**nitpick:** Can you rename this file to `pill_component.html.erb`

Reply:
Good catch. Fixed in 99b2aaa00d86610c8060aca2f5dc2d673736b4ec